jina/proto/jina.proto

```diff
@@ -140,10 +140,9 @@ message DocumentProto {
  */
 message RouteProto {
     string pod = 1; // the name of the BasePod
-    string pod_id = 2; // the id of the BasePod
-    google.protobuf.Timestamp start_time = 3; // receiving time
-    google.protobuf.Timestamp end_time = 4; // sending (out) time
-    StatusProto status = 5; // the status of the execution
+    google.protobuf.Timestamp start_time = 2; // receiving time
```

Comment:
This is not accurate anymore, we should explicitly define what a route means. Sending from gateway time, receiving in gateway time, etc ...